Main/Src/objects/compliancequest__SQX_Complaint__c/compliancequest__SQX_Complaint__c.object-meta.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where are the added fields in PR?

Reply:
Fields has been committed, please check the same.